src/components/PositionedOverlay/PositionedOverlay.tsx

```diff
@@ -298,7 +317,7 @@ function windowRect() {
     top: window.scrollY,
     left: window.scrollX,
     height: window.innerHeight,
-    width: window.innerWidth,
+    width: document.body.clientWidth,
```

Comment:
window.innerWidth includes the scrollbars, We don't want this when aligning from the right and it's shouldn't impact the left.

Reply:
TIL! Will this change alone fix the measuring then and the other stuff in this PR is just a nice to have?

Reply:
No, the right positioning is what is needed. I'll write it in the description.